runtime/compiler/runtime/JITServerAOTCache.cpp

```diff
@@ -75,13 +112,32 @@ ClassSerializationRecord::ClassSerializationRecord(uintptr_t id, uintptr_t class
    memcpy(_name, J9UTF8_DATA(J9ROMCLASS_CLASSNAME(romClass)), _nameLength);
    }
 
+ClassSerializationRecord::ClassSerializationRecord() :
+   AOTSerializationRecord(0, 0, AOTSerializationRecordType::Class),
+   _classLoaderId(0), _hash(NULL), _romClassSize(0),
+   _nameLength(0)
+   {
+   }
+
+bool
+ClassSerializationRecord::isValid() const
```

Comment:
This (and identical method for other record types) can be factored out into a common method `AOTSerializationRecord::isValid(AOTSerializationRecordType type)`.